#150704019 Add travis badge to README

briankabiro opened the pull request:
#### What does this PR do?
Adds the travis badge to the README file

#### Description of Task to be completed?
Added the travis badge to the readme file so that we can track the project build of the repo.

#### Any background context you want to provide?
The build passes on Travis CI and the badge will update when the branch is merged.

#### What are the relevant pivotal tracker stories?
#150704019

Commit:
[Feature #150704019] Add travis badge to README

Added the travis badge to the readme file so that we can track the project build of the repo

collinmutembei commented on README.md:
```diff
@@ -1,4 +1,5 @@
 # healthchecks
+[![Build Status](https://travis-ci.org/andela/healthchecks-f-society.svg?branch=develop)](https://travis-ci.org/andela/healthchecks-f-society)
```
Remove the query parameter specifying the `develop` branch. So as to have the badge update based off the last build.

i.e 
```markdown
[![Build Status](https://travis-ci.org/andela/healthchecks-f-society.svg)](https://travis-ci.org/andela/healthchecks-f-society)
```

collinmutembei reviewed (commented):
Attend to this, and all is good.

Commit:
[Feature #150704019] Update travis badge to README

Updated the travis badge to the readme file so that we can track the lastest project builds of the repo
